### DISCUSSION TITLE
Generally, has anyone come up with a solution to support things like autopairs-nvim and endwise-style alongside blink (similar to how nvim-cmp works?)

### DISCUSSION THREAD

**Discussion by megalithic** [saghen/blink.cmp#157] [Q&A] [answered]
Generally, has anyone come up with a solution to support things like autopairs-nvim and endwise-style alongside blink (similar to how nvim-cmp works?)

**Comment by benfrain** [comment, marked as answer]
This is fixed for me, just needed to add 

```
accept = { auto_brackets = { enabled = true } },
```

To my config

**Reply by saghen** [reply]
As of the configuration rework coming in `v0.7.0`, this is now `completion = { accept = { auto_brackets = { enabled = true } } }`

**Reply by majordoobie** [reply]
Is this option enabled by default? Seems like it is according to the docs? I can't tell.

**Reply by thecodebreeze** [reply]
Just to be sure, @saghen, would this mean that nvim-autopairs is not necessary if I enable this?
